Resources/Prototypes/Catalog/Cargo/cargo_food.yml

```diff
@@ -33,7 +33,7 @@
   icon:
     sprite: Structures/Storage/kvass.rsi
     state: kvass
-  product: CrateFoodKvassTank
+  product: KvassTankFull
```

Comment:
Mqke sure you remove the old CrateFoodKvassTank, as itd be unused content after this pr.

Reply:
CrateFoodKvassTank has been removed